packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -58,8 +59,7 @@ describe('common/sqlite', () => {
     expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
-  test('basic introspection with invalid --url', async () => {
-    ctx.fixture('introspection/sqlite')
+  test('basic introspection with invalid --url if schema is unspecified', async () => {
```

Comment:
why this change and removal of fixtures on that existing test?

Reply:
Because the schema is automatically inferred if `prisma/prisma.schema` is available in the current location. We wanted to test the case in which the `--url` flag is executed without any knowledge of a prisma schema.

Reply:
And the previously existing test is useless and is not needed any more?